src/app/campaign/availability/availability.component.scss

```diff
@@ -34,6 +34,16 @@ section {
     border-top: $row-divider-border;
   }
 
```

Comment:
Both `&.` and `& + &.` rules because we have an `& + & rule` with a border-top I needed to override, aint specificity great